resources/views/welcome.blade.php

```diff
@@ -10,31 +10,23 @@
     <!-- Fonts -->
     <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700&display=swap" rel="stylesheet">
 
-    <!-- Styles -->
-    <link rel="stylesheet" href="{{ mix('css/app.css') }}">
-
-    <!-- Scripts -->
-    <script src="{{ mix('js/app.js') }}" defer></script>
-
-    @if (config('app.env') == 'local')
-        <script src="http://localhost:35729/livereload.js"></script>
-    @endif
+    @vite(['resources/css/app.css', 'resources/js/app.js'])
 </head>
 
 <body class="antialiased">
     <div
-         class="items-top relative flex min-h-screen justify-center bg-gray-100 py-4 dark:bg-gray-900 sm:items-center sm:pt-0">
+        class="items-top relative flex min-h-screen justify-center bg-gray-100 py-4 dark:bg-gray-900 sm:items-center sm:pt-0">
         @if (Route::has('login'))
             <div class="fixed top-0 right-0 hidden px-6 py-4 sm:block">
                 @auth
                     <a href="{{ url('/dashboard') }}"
-                       class="text-sm text-gray-700 underline dark:text-gray-500">Dashboard</a>
+                        class="text-sm text-gray-700 underline dark:text-gray-500">Dashboard</a>
```

Comment:
Todo: I need to work on the blade formatting. For now, I want to revert these whitespace changes.